tests/Go/PhpUnit/ReflectionClass.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/*
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2011, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\PhpUnit;
+
+use Go\Instrument\PathResolver;
+use Go\ParserReflection\ReflectionClass as BaseReflectionClass;
+use Go\ParserReflection\ReflectionEngine;
+use Go\ParserReflection\ReflectionFile;
+
+final class ReflectionClass extends BaseReflectionClass
```

Comment:
Maybe rename it to the `ProxyReflectionClass`? To prevent collision in names with original `ReflectionClass`